umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/spatial.clj

```diff
@@ -108,27 +114,25 @@
      :HorizontalSpatialDomain      (parse-horizontal-spatial-domain doc)
      :VerticalSpatialDomains       (spatial-conversion/convert-vertical-spatial-domains-from-xml
                                     (select spatial "VerticalSpatialDomain"))
-     :OrbitParameters              (as->(fields-from (first (select spatial "OrbitParameters"))
-                                                     :SwathWidth
-                                                     :Period
-                                                     :InclinationAngle
-                                                     :NumberOfOrbits
-                                                     :StartCircularLatitude) op
-                                        ;; Add assumed units for the corresponding fields.
-                                        ;; Replace :Period with :OrbitPeriod.
-                                        (if (:SwathWidth op) 
-                                          (assoc op :SwathWidthUnit "Kilometer")
-                                          op)
-                                        (if (:Period op) 
-                                          (assoc op :OrbitPeriod (:Period op)
-                                                    :OrbitPeriodUnit "Decimal Minute")
-                                          op)
-                                        (if (:InclinationAngle op)
-                                          (assoc op :InclinationAngleUnit "Degree")
-                                          op)
-                                        (if (:StartCircularLatitude op)
-                                          (assoc op :StartCircularLatitudeUnit "Degree")
-                                          op)
-                                        (dissoc op :Period))}
+     :OrbitParameters              (when-let [[o] (select spatial "OrbitParameters")]
```

Comment:
same comment about o

Reply:
same change - renamed to orbit-parameters